_posts/2019-08-08-write-a-new-post.md

```diff
@@ -472,13 +472,50 @@ You can also specify additional attributes for the embedded video file. Here is
 - `autoplay=true` - video automatically begins to play back as soon as it can
 - `loop=true` - automatically seek back to the start upon reaching the end of the video
 - `muted=true` - audio will be initially silenced
+- `extnames` - specify the extensions of additional video formats separated by `|`. Ensure these files exist in the same directory as your primary video file.
 
 Consider an example utilizing all of the above:
 
 ```liquid
-{% include embed/video.html src='video.mp4' poster='poster.png' title='Demo video'
-   autoplay=true loop=true muted=true %}
+{%
+  include embed/video.html
+  src='/path/to/video/video.mp4'
+  extnames='ogg|mov'
+  poster='poster.png'
+  title='Demo video'
+  autoplay=true
+  loop=true
+  muted=true
+%}
+```
+> It's not recommended to host video files in `assets` folder as they cannot be cached by PWA and may cause issues.
+> Instead, use CDN to host video files. Alternatively, use a separate folder that is excluded from PWA (see `pwa.deny_paths` setting in `_config.yml`).
+{: .prompt-warning }
+
+## Audio
+
+### Basic Audio Embedding
+
+To embed an audio file, use the following code snippet:
+
+```liquid
+{% include embed/audio.html src='/path/to/audio/audio.mp3' title='Demo audio' %}
```

Comment:
I would review and update documentation for audio section to looks closer to the documentation for video. It makes it easier to read one after another when they look similar.